### DISCUSSION TITLE
Installation on Centos 8

### DISCUSSION THREAD

**Discussion by yurkessh** [madMAx43v3r/chia-plotter#281] [Q&A] [answered]
Hi all,

Great thanks for an awesome implementation - already tested on Ubuntu 20.04 - works like a charm.

I do have a second plotter/farmer in a server under Centos 8 though, which has been doing a decent job for now, but since it has plenty of RAM and cores, I wanted to give a try.
I wasn't able to compile on CentOS 8 since I couldn't satisfy some of the dependencies (gmp-static, libsodium-static and also a different approach to devtools...).

Would you please point me in a right direction or probably you have plans for CentOS 8 as well?
Would be really much appreciated!

BR,
Yuriy

**Comment by KotaroYasukawa** [comment]
Centos 8 doesn't provide gmp-static package. So I compiled gmp library with enable-static option by myself and now it works well.

**Comment by yurkessh** [comment, marked as answer]
Thanks a lot Yasukawa san ヾ(*´ー`*)ﾉ゛

In fact it's been to early to fire out this discussion since I have missed to read the Issues (only read through all Discussions) while looking for a solution.
I was finally able to solve the problem following this suggestion, which explains, how to do it in details:
https://github.com/madMAx43v3r/chia-plotter/issues/176#issuecomment-859318211

Big thanks to @spencecopper!
